graphlearn/src/service/dist/grpc_channel.cc

```diff
@@ -106,7 +113,13 @@ Status GrpcChannel::CallDagValues(const DagValuesRequestPb* req,
 }
 
 Status GrpcChannel::CallStop(const StopRequestPb* req, StatusResponsePb* res) {
-  if (broken_) {
+  // TODO(tao): do we need such a check ?
+  //
+  // if (stopped_.load()) {
+  //   return Status::OK();
+  // }
+  stopped_.store(true);
```

Comment:
The `stopped_` flag is to record the channel stopped state that I can see, so it seems that we should move the setting of `stopped_` after `HandleStop`

Reply:
Resolved.